src/Umbraco.Web/Editors/AuthenticationController.cs

```diff
@@ -141,5 +146,20 @@ public HttpResponseMessage PostLogout()
         {           
             return Request.CreateResponse(HttpStatusCode.OK);
         }
+
+        // From: http://stackoverflow.com/a/740431/5018
+        protected string GetIPAddress(HttpContextBase httpContext)

```

Comment:
Did this get removed again? I can't seem to find it in AuthenticationController, or HttpContextBase